### ISSUE TITLE
How to make new tab always open the site or function that you need ? 

### ISSUE THREAD

**Issue by ruiite** [minbrowser/min#964]
i just really like to use new tab with opening my first bookmarks and just from new tab immediately open bookmark . Is it possible to make new tab action always to open list of bookmarks ?

**Comment by PalmerAL** [comment]
Are you thinking that the new tab should show your bookmarks list, or that there should be a button to open a list of bookmarks simultaneously?

One thing you could do is keep a task open with your bookmark set in it, and then switch to it whenever you want to use those.

**Comment by ruiite** [comment]
> Are you thinking that the new tab should show your bookmarks list, or that there should be a button to open a list of bookmarks simultaneously?
> 
> One thing you could do is keep a task open with your bookmark set in it, and then switch to it whenever you want to use those.

Yes i wanted to make new tab show my bookmark list . So as i understand now it can't be done ?(

**Comment by PalmerAL** [comment]
One other question: are your bookmarks usually the sites that you visit the most frequently, or not?

**Comment by ruiite** [comment]
> One other question: are your bookmarks usually the sites that you visit the most frequently, or not?

yes usually visit sites in bookmarks.

**Comment by PalmerAL** [comment]
I think there's a couple of different issues here (but please let me know if this isn't right):

* The list of suggested sites doesn't appear when you first start the browser (it only appears for tabs you create later).
* Sometimes, the list of suggested sites is different from your most frequently-visited sites.
* There's no way to see more than 4 suggestions.

1 is hard to fix because the history database isn't available until shortly after startup, and showing the suggestions after a delay makes startup seem slower. 2 is definitely fixable; I'm going to experiment with some different ways of finding suggestions that should make it match the top sites list more closely (while still reacting somewhat to open tabs). 3 is probably fixable as well if we add a "show more" button.

We could add an option to use bookmarks instead of most-visited sites, but I think most-visited is going to be more useful for most people, and as long as your bookmarked sites are also your most-visited, that should still work.

**Comment by PalmerAL** [comment]
2 is done in a72e0e9dbcf9916f103f661e4eff7bb5daeaf596.

Assuming your most-visited sites are the same as your bookmarks, this should work fairly well now, but if they don't it will still be an issue.
